### PR TITLE
Release `v1.0.0b1`

### PR DESCRIPTION
**Pull request by sphuber**

### CODE REVIEW

File: setup.json [hunk 1]
```diff
@@ -1,221 +1,220 @@
 {
-  "name": "aiida-core",
-  "version": "1.0.0a4",
-  "url": "http://www.aiida.net/",
-  "license": "MIT License",
-  "author": "The AiiDA team",
-  "author_email": "developers@aiida.net",
-  "include_package_data": true,
+  "name": "aiida-core",
```
Comment (ltalirz) [inline setup.json]: hm... what is adding the space here? is this the `validate_consistency.py` script?

Comment (sphuber) [reply setup.json]: Yes this was done automatically, I think it is the json dumper that is being used

Comment (ltalirz) [reply setup.json]: we could use [this](https://stackoverflow.com/a/16311587)

Comment (sphuber) [reply setup.json]: If it bothers you, I am fine with adding it

Comment (ltalirz) [reply setup.json]: yes, please!

Comment (sphuber) [reply setup.json]: Everything else ok? Like to do all necessary changes in one go, so we don't have to run too many builds

Comment (ltalirz) [reply setup.json]: I looked through everything and did not see any other problems

Comment (sphuber) [reply setup.json]: Done and done

**Pull request merged**